apps/passport-client/src/localstorage.ts

```diff
@@ -187,6 +187,7 @@ function cleanUpDeprecatedStorage(): void {
     window?.localStorage?.removeItem("offline_tickets");
     window?.localStorage?.removeItem("checked_in_offline_devconnect_tickets");
     window?.localStorage?.removeItem("credential-cache-multi");
+    window?.localStorage?.removeItem("credential-cache-multi-email");
```

Comment:
new credential (sort of) format means we need to expire all old credentials, this is 1/2 of that process, other half is renaming the localstorage key of the cache in `CredentialManager`